modules/@apostrophecms/ui/ui/apos/mixins/AposAdvisoryLockMixin.js

```diff
@@ -93,21 +93,14 @@ export default {
     // detect lock errors with `isLockedError` and call this method. The rest of the time,
     // it is called for you.
     async showLockedError(e) {
-      if (e.body.data.me) {
-        // We use an alert because it is a clear interruption of their
-        // work, and because a notification would appear in both windows
-        // if control was taken by the same user in another window,
-        // which would be confusing.
-        await apos.alert({
-          heading: 'You Took Control in Another Window',
-          description: 'You took control of this document in another tab or window.'
-        });
-      } else {
-        await apos.alert({
-          heading: 'Another User Took Control',
-          description: 'Another user took control of the document.'
-        });
-      }
+      // We use an alert because it is a clear interruption of their
```

Comment:
I don't think we need this comment, or rather, it should just be "Alert user that the document is being edited in another window. Appears in all windows...." for example. We don't need to justify that its not a notification.